src/core/model.js

```diff
@@ -624,9 +624,11 @@ count: ${this.stats.profileFrameCount}`
       instanced = attribute.instanced;
       size = attribute.size;
       if (attribute.externalBuffer) {
-        value = attribute.externalBuffer.data;
-        bytes = attribute.externalBuffer.bytes;
-        verts = bytes / value.BYTES_PER_ELEMENT;
+        if (attribute.externalBuffer.data || isWebGL2(this.gl)) {
```

Comment:
Nit: Recommend "logically" separating the conditionals to make it more clear that we don't hit the webgl2 path.

```
value = externalBuffer.data;
if (!value && isWebGL2()) {
  value = attribute.externalBuffer.getData({length: 10});
}
verts = value ? byte / values.BYTES_PER_ELEMENT : ???; // looks like this case is not handled.
```

Also looks like you can combine verts calculation with else statement below

Reply:
>verts = value ? byte / values.BYTES_PER_ELEMENT : ???; // looks like this case is not handled.

All of them are initialized to default value `NA`, and this code sets them when it can get valid values.